script.js

```diff
@@ -1,12 +1,116 @@
-//You can edit ALL of the code here
+//TV Show Project
+
+const allEpisodes = getAllEpisodes();
+const episodeContainer = document.getElementById("cardContainer");
+const rootElem = document.getElementById("root");
+const searchInput = document.getElementById("searchEpisode");
+const episodesCount = document.getElementById("searchResult");
+const viewAllEpisodesButton = document.querySelector("#viewAllEpisodes");
```

Comment:
Is there a particular reason you used querySelector here when you used getElementById on your other variables?